modules/acm_sku_stock/acm_sku_stock.install

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @file
+ * Installation and update functions for SKU Stock.
+ */
+
+/**
+ * Implements hook_schema().
+ */
+function acm_sku_stock_schema() {
```

Comment:
Why did you decide to use just custom table over custom entity?

Reply:
Entity comes with lot of overhead which we don't want for stock